Updated the lutris install

mandle420 opened the pull request:
just did a fresh install. some changes needed to be made as the current instructions were out of date, also added time saver tip for those who have the game files backed up.

ChaosRifle commented:
lines 41-43: the OB IS still downloadable. openbeta is just a name string that if the version exists, will be downloaded by the installer. if not, it uses stable. the ob installer works just fine. you can edit the openbeta installer to thisisarandomname243 and it will install under that directory and naming too. mostly the ob installer is used due to the deps it references avoiding winetricks.

lines 46-54: these are dependant on the installer used, and is still relevant. though it should be documented that it is for specific installs. namely, for some of the non-ob installers.

lines 58-67: why does this get relocated? 

lines 81-82: sound removal as its redundant with lines 74-78 but we should probably mention and supply an options.lua with the no-launcher flag enabled as another workaround.

mandle420 commented:
line 41-43/46-54: indeed, the ob is still downloadable. The last time I tried maybe I had network issues? I dunno.  As for the deps, the ED version installs them, I assume same as the ob version. I don't even think winetricks is needed for deps at all anymore, unless you're doing wine/proton. 

58-67:  it makes more sense to make sure the game works in 2d mode before moving on to vr.

line 81,82: huh??? maybe I missed something, cuz I don't see that in the commit history...

Made some changes re:above.

ChaosRifle commented:
the deps are different for ed and ob. ed uses "corefonts vcrun2017 dxdiag xact_x64 xmllite win10" and OB uses "arial d3dcompiler_47 win10"
not tried the standard installer in the last month, but it ran d3dcomp43, which caused issues. xact64 is however installed by default now, yes. vcrun2017 can in some cases cause issues and is not defualt installed. removal is not great for windows deps without bottles.
pretty sure you or ED had issues at the time, as that is the version basically everyone is using in the dcs matrix chat, and the few that dont had to add the deps from the OB installer. several users in the last month or two have had to use winetricks to fix the non-ob. a ton of windows users still use open-beta too.


re 58-67, i guess thats fair enough, though one could argue that if you pancake just skip that step so it makes sense to have it there as optional for vr support, as its in the order it should be performed in. semantics i suppose, but makes more sense for vr users. (im not one myself, so maybe one of them could chime in)


for 81:82, check the diffs at https://github.com/budderpard/DCS_Standalone_on_linux/pull/5/files

mandle420 commented:
ok, fixed it and added vr on lutris. only way I could get it working. used open beta, proton-exp, and had to setup a couple other things, but it's workin! finally... Let me know what other changes you think.

ChaosRifle commented on README.md:
```diff
@@ -38,34 +38,23 @@ Unfortunately, workarounds easily get buried there and the OG of this Doc is out
-I used the latter labeled Standalone Open Beta version, but they both should work
-as DCS no longer has an "OpenBeta" and I believe that the install scripts may be borked.
-this will download, the DCS updater and you should install the game
+I used the latter labeled Standalone Open Beta version, but they both should work as DCS no longer has an "OpenBeta" and I believe that the install scripts may be borked. this will download, the DCS updater and you should install the game.
```
```suggestion
I used the latter labeled Standalone Open Beta version, but they both should work as DCS no longer has an "OpenBeta" Using non-OB installer requires d3dcompiler_47.dll be added to the prefix. Currently DCS has no OB branch, so installing with OB will just install the stable branch. this will download, the DCS updater and you should install the game.
```

ChaosRifle commented on README.md:
```diff
@@ -38,34 +38,23 @@ Unfortunately, workarounds easily get buried there and the OG of this Doc is out
 
 ### Getting it working with Lutris
+This step should be by the lutris installer, but check just in case.
+Start the game once first to create the prefix(here for an explination of what a prefix is), then use lutris's Winetricks to add these .dll and font.
```
```suggestion
Start the game once first to create the ([prefix](https://wiki.archlinux.org/title/Wine#WinePREFIX), then use lutris's Winetricks to add these .dll and font.
```

ChaosRifle commented on README.md:
```diff
@@ -38,34 +38,23 @@ Unfortunately, workarounds easily get buried there and the OG of this Doc is out
-![Winetricks](images/Winetricks.png)
+
+you can open Winetricks in lutris by clicking on DCS Do not open, 1 LMB click only, then click on the Wine glass at the bottom of the screen
+Winetricks
```
```suggestion
![Winetricks](images/Winetricks.png)
```

ChaosRifle reviewed (commented):
added a a change to denote status of OB
reverted accidental removal of linked images and information.
truncated old informational link to be cleaner.

mandle420 commented:
Thanks for those fixes. Cheers.

mandle420 commented:
closing this for now. may fork and update the newest version for vr envision w/ monado
